.travis.yml

```diff
@@ -85,38 +42,6 @@ matrix:
         - npm test -- --coverage
     - stage: large
       if: type != cron
-    - language: python
-      python: 2.7
-      if: type != cron
-      install:
-        - INSTALL_LARGE_PYTHON_DEPS=true source ./travis/install-common-deps.sh
-      script:
-        - ./travis/run-large-python-tests.sh
-    # Run Windows tests in the "large" test builder so that we don't spend Travis executor
-    # time running Windows tests if Python 3 small tests fail.
-    - os: windows
-      name: "Windows"
-      if: type != cron
-      language: sh
-      before_install:
-        - choco install python3 --version=3.7.4
-      install:
-        - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
-        - pip install -r dev-requirements.txt
-        - pip install -r travis/small-requirements.txt
-        - pip install -e .
-      script:
-        - pytest --verbose --ignore=tests/h2o --ignore=tests/keras --ignore=tests/pytorch --ignore=tests/pyfunc --ignore=tests/sagemaker --ignore=tests/sklearn  --ignore=tests/spark --ignore=tests/tensorflow --ignore=tests/keras_autolog --ignore=tests/tensorflow_autolog --ignore tests/azureml --ignore tests/onnx --ignore tests/projects --ignore=tests/xgboost --ignore=tests/lightgbm tests
-    # Run small Python 2.7 tests in the "large" test builder so that we don't spend Travis executor
-    # time running Python 2.7 tests if Python 3 small tests fail.
-    - language: python
```

Comment:
Large tests are run only on 2.7 (sad) So please don't remove it

Reply:
IRL discussion, large tests are run after this removal